cpp/src/text/subword/bpe_tokenizer.cuh

```diff
@@ -21,39 +21,96 @@
 #include <hash/hash_allocator.cuh>
 
 #include <cudf/column/column.hpp>
+#include <cudf/column/column_device_view.cuh>
 #include <cudf/hashing/detail/murmurhash3_x86_32.cuh>
+#include <cudf/strings/string_view.cuh>
 
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/device_uvector.hpp>
 #include <rmm/mr/device/polymorphic_allocator.hpp>
 
+#include <cuco/operator.hpp>
```

Comment:
Is this inclusion necessary? If so, I probably need to add it to the top-level `static_map.cuh` header as well.